doc/mgr/diskprediction.rst

```diff
@@ -0,0 +1,113 @@
+=====================
+DiskPrediction plugin
+=====================
+
+Disk Prediction plugin is used to collect disk information from Ceph OSD and
+send DiskPrediction server. The disk information includes the following.
+
+- Ceph status
+- I/O operations
+- I/O bandwidth
+- OSD status
+- OSD physical disk smart data
+- Storage utilization
+
+Enabling
+========
+
+Run the following command to enable *diskprediction* module in the Ceph
+environment:
+
+::
+
+    ceph mgr module enable diskprediction
+
+Connection settings
+-------------------
+
+The connection settings can be configured on any machine with the proper cephx
+credentials; they are usually the monitor node with client.admin keyring.
+Run the following command to set up connection between Ceph system and
+DiskPrediction server.
+
+::
+
+    ceph diskprediction config-set <diskprediction_server> <diskprediction_user> <diskprediction_password>
+	
+
+The <diskprediction_server> parameter is DiskPrediction server name, and it
```

Comment:
Please enclose commandline parameters in the text block in double backticks so they are rendered in a monospace font.

Reply:
Got it, Thanks.